Adafruit_MQTT.cpp

```diff
@@ -265,17 +265,19 @@ uint16_t Adafruit_MQTT::readFullPacket(uint8_t *buffer, uint16_t maxsize,
   } while (encodedByte & 0x80);
 
   DEBUG_PRINT(F("Packet Length:\t"));
-  DEBUG_PRINTLN(value);
+  DEBUG_PRINTLN(fixedHeaderLength + remainingLength);
```

Comment:
NIT: this changes the meaning from the debug message before, but I actually like it better now. ;)

Reply:
According to the spec, the packet length includes the fixed header length, variable header length and payload length. The remaining length is the variable header length plus payload length. The previous code is inaccurate. Either say `"Remaining Length"` or include the `fixedHeaderLength` in the output.

Reply:
I'm ok with this output.